tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_marketplace_download_execute_map_app.py

```diff
@@ -0,0 +1,90 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import time
+
+from marionette_driver import Wait, By
+
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.homescreen.app import Homescreen
+from gaiatest.apps.marketplace.app import Marketplace
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+
+
+class TestSearchMarketplaceAndInstallApp(GaiaImageCompareTestCase):
+    """
+    Download bing map from marketplace and open it
+    """
+    app_search = 'Bing Maps :packaged'
+    app_title = 'Bing Maps'
+    wait_until_draw_done = 5
```

Comment:
Maybe this should be called draw_done_time, instead of wait_until_draw_done? wait_until_draw_done makes it sound like it's a function or method.